src/core/lib/iomgr/ev_epollex_linux.cc

```diff
@@ -1230,17 +1263,15 @@ static grpc_error* pollset_add_fd_locked(grpc_pollset* pollset, grpc_fd* fd) {
       error = pollset_transition_pollable_from_empty_to_fd_locked(pollset, fd);
       break;
     case PO_FD:
-      gpr_mu_lock(&po_at_start->owner_fd->orphan_mu);
-      if ((gpr_atm_no_barrier_load(&pollset->active_pollable->owner_fd->refst) &
-           1) == 0) {
-        error =
-            pollset_transition_pollable_from_empty_to_fd_locked(pollset, fd);
+      gpr_mu_lock(&po_at_start->owner_orphan_mu);
+      if (po_at_start->owner_orphaned) {
+        pollset_transition_pollable_from_empty_to_fd_locked(pollset, fd);
```

Comment:
add `error = `?

Reply:
this is a great catch!. thanks Yang.

Reply:
(I am not even sure how i missed this. I don't remember touching this line :)